src/components/ec-panel/ec-panel.vue

```diff
@@ -36,24 +49,37 @@ export default {
   },
   model: {
     prop: 'show',
-    event: 'close',
+    event: 'show-panel',
   },
   props: {
     show: {
       type: Boolean,
       default: false,
     },
+    isBackEnabled: {
```

Comment:
I wouldn't make this as a prop. I would convert it to a computed prop and check if $listeners['back'] exists inside. Then everything you have to do to enable back functionality is to `<ec-panel @back="doSomething()">`

Reply:
Smarter approach. Done.